lookupapi/urls.py

```diff
@@ -57,7 +57,7 @@
 urlpatterns = [
     path('attributes/people', views.PersonFetchAttributes.as_view(), name='person-attributes'),
     path('people', views.PersonList.as_view(), name='person-list'),
-    path('people/self', views.PersonSelf.as_view(), name='person-self'),
+    path('people/token/self', views.PersonSelf.as_view(), name='person-token-self'),
```

Comment:
Why is this not handled like the mock or crsid schemes?

Reply:
From deiscussion: change this use the normal scheme/identifier endpoint.

Reply:
done